zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -239,6 +239,9 @@ pub(crate) fn get_unspent_transparent_output(
     result
 }
 
+/// Returns spendable transparent outputs that have been received by this wallet at the given
+/// transparent address, as outputs of transactions in blocks mined at a height less than or
+/// equal to the provided `max_height`.
```

Comment:
Yes, I will fix this.

Reply:
The documentation on main here is wrong (it still refers to `max_height` and filtering to respect number of confirmations: https://github.com/zcash/librustzcash/blob/34cf6d286a201caf7dfee29fbee867d547afc486/zcash_client_backend/src/data_api.rs#L680-L685

What should it be?

Reply:
```suggestion
/// Returns the list of spendable transparent outputs received by this wallet at `address`
/// such that, at height `target_height`:
/// * the transaction that produced the output had or will have at least `min_confirmations`
///   confirmations; and
/// * the output is unspent as of the current chain tip.
///
/// An output that is potentially spent by an unmined transaction in the mempool is excluded
/// iff the spending transaction will not be expired at `target_height`.
///
/// This could, in very rare circumstances, return as unspent outputs that are actually not
/// spendable, if they are the outputs of deshielding transactions where the spend anchors have
/// been invalidated by a rewind. There isn't a way to detect this circumstance at present, but
/// it should be vanishingly rare as the vast majority of rewinds are of a single block.
```